Create ui-c8y-1021-44-0-allowing-packages-to-get-archived-7396-graft-release-cd-8174.md (#2964)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-44-0-allowing-packages-to-get-archived-7396-graft-release-cd-8174.md

---
date: ""

> title: Allow extension packages to be archived

Reviewer: ```suggestion
title: Extension packages can now be archived
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61341
version: 1021.44.0
---

> In the past, extension packages could not be archived in the system which led to challenges with managing obsolete or unused packages. With this change, it is now possible to archive packages which are no longer maintained. This allows to mark packages as out of maintenance, while still being able to access archived packages if necessary.

Reviewer: ```suggestion
In the past, extension packages could not be archived in the system which led to challenges with managing obsolete or unused packages. With this change, it is now possible to archive packages which are no longer maintained. Packages can be marked as out of maintenance, while still being able to access archived packages if necessary.
```

> In the past, extension packages could not be archived in the system which led to challenges with managing obsolete or unused packages. With this change, it is now possible to archive no longer maintained packages. This allows extension package maintainers to mark their package as out of maintenance, while still being able to access archived packages if necessary.

Reviewer: ```suggestion
In the past, extension packages could not be archived in the system which led to challenges with managing obsolete or unused packages. With this change, it is now possible to archive packages which are no longer maintained. This allows to mark packages as out of maintenance, while still being able to access archived packages if necessary.
```
